app/assets/stylesheets/course/user_invitations.scss

```diff
@@ -0,0 +1,16 @@
+.course-user-invitations {
+  &.new {
+    div.methods {
+      ul.nav {
+        margin-bottom: 1em;
+      }
+
+      div.tab-pane#registration_code .registration_code {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Avoid qualifying class selectors with an element.<br>Selector `registration_code` should be written in lowercase with hyphens